kfs-core/src/main/java/org/kuali/kfs/fp/batch/service/impl/ProcurementCardCreateDocumentServiceImpl.java

```diff
@@ -95,7 +94,23 @@
 import org.kuali.rice.krad.workflow.service.WorkflowDocumentService;
 import org.springframework.transaction.annotation.Transactional;
 
-
+import edu.arizona.kfs.fp.businessobject.ProcurementCardLevel3Add;
```

Comment:
I attempted to create an edu.arizona version of service class ProcurementCardCreateDocumentServiceImpl but the changes were so pervasive, that almost every single method needed to have a signature change override. I decided it was cleaner (in this case) to make the changes/additions in base code.